config/locales/en.yml

```diff
@@ -1,33 +1,19 @@
-# Files in the config/locales directory are used for internationalization
-# and are automatically loaded by Rails. If you want to use locales other
-# than English, add the necessary files in this directory.
-#
-# To use the locales, use `I18n.t`:
-#
-#     I18n.t 'hello'
-#
-# In views, this is aliased to just `t`:
-#
-#     <%= t('hello') %>
-#
-# To use a different locale, set it with `I18n.locale`:
-#
-#     I18n.locale = :es
-#
-# This would use the information in config/locales/es.yml.
-#
-# The following keys must be escaped otherwise they will not be retrieved by
-# the default I18n backend:
-#
-# true, false, on, off, yes, no
-#
-# Instead, surround them with single quotes.
-#
-# en:
-#   'true': 'foo'
-#
-# To learn more, please read the Rails Internationalization guide
-# available at http://guides.rubyonrails.org/i18n.html.
-
 en:
-  hello: "Hello world"
+  default_title: "SearchWorks"
+  opensearch_description: "SearchWorks"
```

Comment:
I think we may want to change this.  This means that the bento app will look like it's searching just SearchWorks (and might be confusing when people have real SearchWorks' opensearch setup).  Perhaps @jvine may have thoughts on this?  Something like `Stanford Libraries`?

## From bento-stage
<img width="184" alt="screen shot 2017-08-09 at 3 01 35 pm" src="https://user-images.githubusercontent.com/96776/29145900-c2c8dd18-7d13-11e7-81ad-017838e0cd62.png">